.github/workflows/tests-cleanup.yml

```
@@ -0,0 +1,77 @@
+name: TestsCleanUp
```

This will show up in the Slack announcement of failure, better to be more specific
```suggestion
name: SQS Transport Cleanup
```